charts/authentik/values.yaml

```diff
@@ -83,7 +85,7 @@ authentik:
     # -- set the postgresql hostname to talk to
     # if unset and .Values.postgresql.enabled == true, will generate the default
     # @default -- `{{ .Release.Name }}-postgresql`
-    host: '{{ .Release.Name }}-postgresql'
```

Comment:
Sorry, this is my code-editor doing things, I didn't even realize these quotes were modified. Let me know if that bothers you.